src/imex.rs

```diff
@@ -115,13 +113,6 @@ pub async fn imex(
     res
 }
 
-async fn cleanup_aborted_imex(context: &Context, what: ImexMode) {
-    if what == ImexMode::ImportBackup {
-        dc_delete_file(context, context.get_dbfile()).await;
-        dc_delete_files_in_dir(context, context.get_blobdir()).await;
```

Comment:
As for blobdir cleanup, worst case is that you start importing some backup, then abort the operation, then import another backup. In this case you get blobs from aborted import in your blobdir. Not a big deal because unreferenced blobs will be cleaned up by housekeeping.